metrics.md

```diff
@@ -1,9 +1,267 @@
 ---
 layout: base
+mathjax: true
 permalink: /metrics/
 ---
 
 # Metrics
+The Solar Forecast Arbiter evaluation framework provides a suite of metrics for evaluating deterministic and probablistic solar forecasts. These metrics are used for different purposes, e.g., comparing the forecast and the measurement, comparing the performance of multiple forecasts, and evaluating an event forecast.
 
-Please download and review the metrics proposal [Word document](../assets/Solar Power Forecasting Evaluation Metrics 20190409.docx) or [pdf](../assets/Solar Power Forecasting Evaluation Metrics 20190409.pdf).
-We are collecting feedback on the proposal through April 19, 2019. Please fill out the survey at [https://www.surveymonkey.com/r/SFIIMetrics](https://www.surveymonkey.com/r/SFIIMetrics). A calendar reminder is available for download [here](../assets/SFA_metrics_reminder.ics).
+
+## Metrics for Deterministic Forecasts
+The following metrics provide measures of the performance of deterministic forecasts. Each metric is computed from a set of $$ n $$ forecasts $$ (F_1, F_2, \dots, F_n) $$ and corresponding observations $$ (O_1, O_2, \dots, O_n) $$.
+
+In the metrics below, we adopt the following nomenclature:
+- $$ n : $$ number of samples
+- $$ F : $$ forecasted value
+- $$ O : $$ observed (actual) value
+- $$ \text{norm} : $$ normalizing factor (with the same units as the forecasted and observed values)
+- $$ \bar{F}, \, \bar{O} : $$ the mean of the forecasted and observed values, respectively
+
+
+### Mean Absolute Error (MAE)
+The absolute error is the absolute value of the difference between the forecasted and observed values. The MAE is defined as:
+
+$$ \text{MAE} = \frac{1}{n} \sum_{i=1}^n  \lvert F_i - O_i \rvert $$
+
+
+### Mean Bias Error (MBE)
+The bias is the difference between the forecasted and observed values. The MBE is defined as:
+
+$$ \text{MBE} = \frac{1}{n} \sum_{i=1}^n (F_i - O_i) $$
+
+
+### Root Mean Square Error (RMSE)
+The RMSE is the square root of the averaged of the squared differences between the forecasted and observed values, and is defined as:
+
+$$ \text{RMSE} = \sqrt{ \frac{1}{n} \sum_{i=1}^n (F_i - O_i)^2 } $$
+
+RMSE is a frequently used measured for evaluating forecast accuracy. It can be interprest
```

Comment:
One more minor thing at this line and this is ready to go.

Reply:
Fixed.